core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Accpetance Test for a Blog Admin to assign roles and change
+ * tag properties from the blog-admin page.
+ */
+
+const createNewUser = require(
+  '../puppeteer-testing-utilities/initializeUsersUtils.js');
+const { closeAllBrowsers } = require(
+  '../puppeteer-testing-utilities/initializeUsersUtils.js');
+
+const ROLE_BLOG_ADMIN = 'blog admin';
+const ROLE_BLOG_POST_EDITOR = 'blog post editor';
+
+let assignRolesToUsersAndChangeTagProperties = async function() {
+  const blogAdmin = await createNewUser.blogAdmin('blogAdm');
+  const superAdmin = await createNewUser.superAdmin('superAdm');
+  await createNewUser.guestUser('guestUsr1', 'guest_user1@example.com');
+  await createNewUser.guestUser('guestUsr2', 'guest_user2@example.com');
+
+  await blogAdmin.assignUserAsRoleFromBlogAdminPage('guestUsr1', 'BLOG_ADMIN');
+  await superAdmin.expectUserToHaveRole('blogAdm', ROLE_BLOG_ADMIN);
+  await blogAdmin.assignUserAsRoleFromBlogAdminPage(
+    'guestUsr2', 'BLOG_POST_EDITOR');
+  await superAdmin.expectUserToHaveRole('guestUsr2', ROLE_BLOG_POST_EDITOR);
```

Comment:
Same thing, not sure why the blog admin can't check these things?

Reply:
this is because blogAdmin cannot visit the /admin page. and here we are checking that roles have being assigned to user from the roles tab in /admin page. users with signin email `testadmin@example.com` can only visit the /admin page

Reply:
OK thanks, see comment above.